### PR TITLE
Improve packet docs, add some notes to README, add unhandled exception logging, and some fixes

### PR DESCRIPTION
**Pull request by TheNathannator**
Did some thorough analysis on the packet logs from GuitarSniffer and from the initial readme, here are my observations.
Also added some important notes for Jaguar guitars and PDP drums, and to try WinPcap instead of Npcap if the receiver isn't detected.

### CODE REVIEW

**Comment by TheNathannator** [comment]
Added unhandled exception logging since someone was getting a crash after clicking Start, with no indication as to why

**Comment by TheNathannator** [comment]
Some robustness improvements (that same person had quite the strange case), as well as more cosmetic/minor changes

File: MainWindow/MainWindow.xaml.cs [hunk 1]
```diff
@@ -1519,15 +1646,47 @@ private bool Read_AutoDetectID()
             // Assume failure
             bool result = false;
 
+            // Check if a device has been selected
+            if (pcapSelectedDevice == null)
+            {
+                uiDispatcher.Invoke((Action)(() =>
+                {
+                    Console.WriteLine("Please select a Pcap device from the Pcap dropdown.");
+                }));
```
Comment (ferzkopp) [inline MainWindow/MainWindow.xaml.cs]: Console.WriteLine() by itself does not need to be called from the uiDispatcher; that is only needed when the UI-elements are changed directly. Recommend to remove the Invoke.

File: README.md [hunk 2]
```diff
@@ -10,15 +10,19 @@ Almost all features for guitars and drums are supported.
...
 - Npcap in WinPcap compatibility mode, or WinPcap
 - USBPcap
 - vJoy or ViGEmBus
 
 ## Installation
 
-1. Install [Npcap](https://nmap.org/npcap/#download) in WinPCap compatibility mode (recommended) or [WinPCap](https://www.winpcap.org/install/default.htm).
+1. Install [WinPCap](https://www.winpcap.org/install/default.htm) (recommended as it seems to work best with Xbox One receivers), or [Npcap](https://nmap.org/npcap/#download) in WinPCap compatibility mode.
+   - Make sure you only install one or the other. Installing both at the same time may cause issues.
+   - If later on your receiver is not detected in the program, try switching which one you installed.
```
Comment (ferzkopp) [inline README.md]: This sounds funky. Please reword.

File: MainWindow/MainWindow.xaml.cs (hunk 1 above)
Comment (TheNathannator) [reply MainWindow/MainWindow.xaml.cs]: Good to know.

File: README.md (hunk 2 above)
Comment (TheNathannator) [reply README.md]: In retrospect, I agree lol

**Comment by TheNathannator** [comment]
Made the requested changes, along with rewording of one other sentence in the readme since I thought it was also kinda funky.

**Pull request merged**